pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -306,7 +306,7 @@ private CompletableFuture<Optional<URL>> internalGetWebServiceUrl(Optional<Servi
             }
             CompletableFuture<Optional<LookupResult>> future =
                     ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(config)
-                    ? loadManager.get().findBrokerServiceUrl(topic, bundle) :
+                    ? loadManager.get().findBrokerServiceUrl(Optional.ofNullable(topic), bundle) :
```

Comment:
I think so.
`@Nullable` indicates to the caller the topic can be null.
Since it can be null we need to use `Optional.ofNullable()` when wrapping it into an Optional.